config/locales/nl.yml

```diff
@@ -37,6 +37,16 @@ nl:
           form:
             toggle_advanced_options: Klik voor toegang tot meta tag instellingen en menu opties
             published_at: Publicatiedatum
+            custom_url: Custom URL
```

Comment:
Not really. We don't have a single word for it, and the Dutch language absorbs many English words. I considered a few translations but they only lead to more confusion and ambiguity.